eventbus/jetstream/sensor/trigger_conn.go

```diff
@@ -257,15 +257,35 @@ func (conn *JetstreamTriggerConn) processMsg(
 		conn.Logger.Errorf("can't get Metadata() for message %+v??", m)
 	}
 
-	defer func() {
-		err = m.AckSync() // this call does the acknowledgment and waits for an acknowledgment from the server (that it received the ack) too
-		if err != nil {
-			errStr := fmt.Sprintf("Error performing AckSync() on message: %v", err)
-			conn.Logger.Error(errStr)
+	done := make(chan bool)
+	go func() {
+		ticker := time.NewTicker(500 * time.Millisecond)
```

Comment:
@juliev0 - how is the jetstream consumer created?

Reply:
See line 132 of this file where getDurableName() is called based on sensor name, trigger name, and dependency name and then passed on line 136 to the PullSubscribe call.

Reply:
(Not sure if that's what you were asking...I guess my answer is really related to the 'consumer group'